ethereumj-core/src/test/java/org/ethereum/datasource/RepoNewTest.java

```diff
@@ -155,31 +155,31 @@ public void testStorage1() throws Exception {
         RepositoryRoot repo = new RepositoryRoot(stateDb, null);
         byte[] addr1 = decode("aaaa");
         repo.createAccount(addr1);
-        repo.addStorageRow(addr1, new DataWord(1), new DataWord(111));
+        repo.addStorageRow(addr1, DataWord.of(1), DataWord.of(111));
```

Comment:
`DataWord.ONE`

Reply:
@mkalinin Here you probably a little bit doebalsa :) This is just a test

Reply:
Ahahaha, maybe just a little bit